bin/od

```diff
@@ -110,7 +122,7 @@ if ($offset1) {
     }
 }
 if (defined($lim) && $lim == 0) {
-    printf("%.8$radix\n", $offset1);
+    printf("%.8$radix\n", $offset1) if $radix;
```

Comment:
Is there ever a case where $radix won't have a value?

Reply:
With this patch, $radix won't be assigned a value if $opt_A is "n"

Reply:
ah, my fault. I read that branch above as `eq` and not `ne`.